Libraries/NativeModules/specs/NativeDialogManagerAndroid.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+import Platform from '../../Utilities/Platform';
+
+type DialogAction = 'buttonClicked' | 'dismissed';
+type DialogButtonKey = 'buttonPositive' | 'buttonNegative' | 'buttonNeutral';
+type DialogOptions = {|
+  +title?: string,
+  +message?: string,
+  +buttonPositive?: string,
+  +buttonNegative?: string,
+  +buttonNeutral?: string,
+  +items?: Array<string>,
+  +cancelable?: boolean,
+|};
+
+export interface Spec extends TurboModule {
+  +getConstants: () => {|
+    buttonClicked: DialogAction,
+    dismissed: DialogAction,
+    buttonPositive: DialogButtonKey,
+    buttonNegative: DialogButtonKey,
+    buttonNeutral: DialogButtonKey,
```

Comment:
Looking through the implementation of `DialogManagerAndroid`, it looks like `buttonPositive`, `buttonNegative`, and `buttonNeutral` are numbers? 

#### DialogModule.java
```Java
  /* package */ static final String ACTION_BUTTON_CLICKED = "buttonClicked";
  /* package */ static final String ACTION_DISMISSED = "dismissed";
  /* package */ static final String KEY_TITLE = "title";
  /* package */ static final String KEY_MESSAGE = "message";
  /* package */ static final String KEY_BUTTON_POSITIVE = "buttonPositive";
  /* package */ static final String KEY_BUTTON_NEGATIVE = "buttonNegative";
  /* package */ static final String KEY_BUTTON_NEUTRAL = "buttonNeutral";
  /* package */ static final String KEY_ITEMS = "items";
  /* package */ static final String KEY_CANCELABLE = "cancelable";

  /* package */ static final Map<String, Object> CONSTANTS = MapBuilder.<String, Object>of(
      ACTION_BUTTON_CLICKED, ACTION_BUTTON_CLICKED,
      ACTION_DISMISSED, ACTION_DISMISSED,
      KEY_BUTTON_POSITIVE, DialogInterface.BUTTON_POSITIVE,
      KEY_BUTTON_NEGATIVE, DialogInterface.BUTTON_NEGATIVE,
      KEY_BUTTON_NEUTRAL, DialogInterface.BUTTON_NEUTRAL);
```

#### DialogInterface.java
```Java
public interface DialogInterface {
    /** The identifier for the positive button. */
    int BUTTON_POSITIVE = -1;

    /** The identifier for the negative button. */
    int BUTTON_NEGATIVE = -2;

    /** The identifier for the neutral button. */
    int BUTTON_NEUTRAL = -3;
```

Reply:
@RSNara,
Thank you for reviewing my PR. I already made the required changes.

Can you take a look on it again, pls?